ui/app/configuration/configuration-controllers.js

```diff
@@ -27,25 +27,21 @@
     'configurationList',
     'PncRestClient',
     function($log, $state, configurationList, PncRestClient) {
-      var self = this;
-      $log.debug('ConfigurationListController >> this=%O, configurationList=%O',
-                 self, configurationList);
-
-      self.configurations = configurationList;
-      self.projects = [];
-
-      angular.forEach(self.configurations, function(configuration){
-
-          PncRestClient.Project.get({
-              projectId: configuration.projectId
-          }).$promise.then(
-            function (result) {
-              if (result) {
-                self.projects.push(result);
-                //console.log(JSON.stringify(self.projects));
-              }
+      var that = this;
+
+      this.configurations = configurationList;
+      this.projects = [];
+
+      angular.forEach(this.configurations, function(configuration) {
```

Comment:
Actually we have a mix throughout the codebase. If you look at configuration-controllers.js it's entirely var that = this (the JS standard). 

Personally I prefer using the `this` keyword everywhere accept where it's necessary to use the saved reference (i.e. for functions that will be used as callbacks).

I'm happy to standardise this but there are a lot of places in the codebase that use the different methods. What do you think?